### DISCUSSION TITLE
Title not showing

### DISCUSSION THREAD

**Discussion by sugengprianto** [hexojs/hexo#4912] [Question / Answer] [answered]
I've created a new post "mypost.md", but the post title doesn't appear in the most recent post.  Please help

![IMG_20220309_200226](https://user-images.githubusercontent.com/55155451/157447868-246e3324-16b5-4fc5-a564-789d4ff2b418.jpg)

**Comment by SukkaW** [comment, marked as answer]
Please use `hexo new post [title]` or `hexo new draft [title]` to create your post or draft. Never create the markdown file by yourself unless you know exactly what you are doing.
